room.cpp

```diff
@@ -169,11 +169,32 @@ void Room::setLastReadEvent(User* user, QString eventId)
     emit lastReadEventChanged(user);
 }
 
-void Room::markMessageAsRead(Event* event)
+void Room::markMessagesAsRead(Timeline::const_iterator end)
```

Comment:
The iterator should not be at the end, but at the element to be marked as read, right?